HLTrigger/Configuration/python/customizeHLTforAlpakavsCUDA.py

```diff
@@ -0,0 +1,911 @@
+                L1TauTrigger = cms.InputTag("hltL1sSingleTau")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('MuXXTauYY'),
+                L1TauTrigger = cms.InputTag("hltL1sBigOrMuXXerIsoTauYYer")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('Mu22Tau40'),
+                L1TauTrigger = cms.InputTag("hltL1sMu22erIsoTau40er")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('DoubleTauJet'),
+                L1TauTrigger = cms.InputTag("hltL1sBigORDoubleTauJet")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('VBFIsoTau'),
+                L1TauTrigger = cms.InputTag("hltL1VBFDiJetIsoTau")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('Mu18TauXX'),
+                L1TauTrigger = cms.InputTag("hltL1sVeryBigORMu18erTauXXer2p1")
+            ),
+            cms.PSet(
+                L1CollectionName = cms.string('DoubleTauLowMass'),
+                L1TauTrigger = cms.InputTag("hltL1sDoubleTauBigORWithLowMass")
+            )
+        ),
+        debugLevel = cms.int32(0),
+        ebInput = cms.InputTag("hltEcalRecHit","EcalRecHitsEB"),
+        eeInput = cms.InputTag("hltEcalRecHit","EcalRecHitsEE"),
+        fractionSumPt2 = cms.double(0.3),
+        graphPath = cms.string('RecoTauTag/TrainingFiles/data/L2TauNNTag/L2TauTag_Run3v1.pb'),
+        hbheInput = cms.InputTag("hltHbhereco"),
+        hoInput = cms.InputTag("hltHoreco"),
+        maxVtx = cms.uint32(100),
+        minSumPt2 = cms.double(0.0),
+        normalizationDict = cms.string('RecoTauTag/TrainingFiles/data/L2TauNNTag/NormalizationDict.json'),
+        pataTracks = cms.InputTag("hltPixelTracksSoAFromCUDA"),
+        pataVertices = cms.InputTag("hltPixelVerticesSoAFromCUDA"),
+        track_chi2_max = cms.double(99999.0),
+        track_pt_max = cms.double(10.0),
+        track_pt_min = cms.double(1.0)
+    )
+    
+    return process
+
+from HLTrigger.Configuration.customizeHLTforAlpaka import customizeHLTforAlpakaPixelRecoLocal
+from HLTrigger.Configuration.customizeHLTforAlpaka import customizeHLTforAlpakaPixelRecoTracking
+from HLTrigger.Configuration.customizeHLTforAlpaka import customizeHLTforAlpakaPixelRecoVertexing
+from HLTrigger.Configuration.customizeHLTforAlpaka import customizeHLTforAlpakaPixelRecoTheRest
+
+def customizeHLTforAlpakavsCUDAPixelReco(process):
+    '''Customisation to introduce the Pixel Local+Track+Vertex Reconstruction in Alpaka and CUDA
+    '''
+
+    process = customizeHLTforAlpakaPixelRecoLocal(process)
+    process = customizeHLTforAlpakaPixelRecoTracking(process)
+    process = customizeHLTforAlpakaPixelRecoVertexing(process)
+    process = customizeHLTforCUDAPixelRecoLocal(process)
+    process = customizeHLTforCUDAPixelRecoTracking(process)
+    process = customizeHLTforCUDAPixelRecoVertexing(process)
```

Comment:
so this would lead to running the Pixel reco in CUDA in the HLT menu, is this what we want, in light of https://github.com/cms-sw/cmssw/pull/43964#issuecomment-1944423984 ?

Reply:
OK, this is protected by https://github.com/cms-sw/cmssw/pull/43964/files#diff-8fc23ee531566af47b6eff1b354bb3db1c4e8f3925413edc3632e417d168b75fR254-R255

Reply:
Yes, exactly.